chadrick-d-dev/silver-nest-api
# Heroku deployment

### What’s this PR do?

* Configures the database with heroku

### Where should the reviewer start?

* `knexfile.js` line 14
* `server.js` line 3
* `procfile` line 1

How should this be manually tested?

* Check heroku

Any background context you want to provide?

* This lesson plan is fundamental to the configuration of heroku: https://frontend.turing.io/lessons/module-4/deploy-to-heroku
